.github/workflows/tests.yml

```
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.3, 7.4, 8.0]
+        php: [8.0]
```

I suggest adding PHP 8.1 as well, it's supported by Laravel 9
```suggestion
        php: [8.0, 8.1]
```

---

Right! Good catch, thanks!